src/bin/cobalt/new.rs

```diff
@@ -436,6 +436,37 @@ fn prepend_date_to_filename(
     Ok(())
 }
 
+fn move_from_drafts_to_posts(
+    config: &cobalt_model::Config,
+    file: &path::Path,
+) -> Result<path::PathBuf> {
+    let mut posts = config.posts.clone();
+    posts.include_drafts = true;
+    let posts = posts.build()?;
+    let drafts = posts.drafts.clone();
+
+    if posts
+        .drafts
+        .map(|d| d.includes_file(&file))
+        .unwrap_or_default()
+    {
+        let drafts = drafts.expect("Should have drafts in this if body");
+        let target = config.source.join("posts").join(
+            file.strip_prefix(drafts.subtree())
+                .expect("Should have subtree as prefix"),
+        );
+        trace!(
+            "post is in `drafts_dir` moving it to `posts` directory: {}",
+            target.to_string_lossy()
```

Comment:
`target.display`
(probably am doing `to_string_lossy` in other places because I didn't know about `display` then)